docs/example-scenario/data/data-warehouse.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Example scenario for data warehousing and analytics in Azure
+description: Use a data warehouse in Azure to consolidate data from multiple sources and optimize data analytics.
+author: alexbuckgit
+ms.date: 08/16/2018
+---
+
+# Example scenario for data warehousing and analytics in Azure
+
+Build a data pipeline to integrate large amounts of data from multiple sources into a cloud-scale analytics platform. This scenario is relevant for:
+* Sales and marketing
+* E-commerce and retail
+* Healthcare
+* Other solutions requiring advanced analysis of large datasets
+
+## The scenario
+
+A sales and marketing company builds incentive programs. These programs reward customers, suppliers, salespeople, and employees. Data is fundamental to these programs, and the company wants to improve the insights gained through data analytics using Azure.
+
+The company needs a modern approach to analysis data, so that decisions are made using the right data at the right time. The company's goals include:
+* Consolidating different kinds of data sources into a cloud-scale platform.
+* Transforming source data into a common taxonomy and structure, to make the data consistent and easily compared.
+* Loading data using a highly parallelized approach that can support thousands of incentive programs, without the high costs of deploying and maintaining on-premises infrastructure.
+* Greatly reducing the time needed to gather and transform data, so you can focus on analyzing the data.
+
+## Potential use cases
+
```

Comment:
"In addition to the other industries we've mentioned above, the design patterns you see demonstrated here could be used for any customer who's looking to do the following"

Reply:
In the spirit of minimizing boilerplate language and increasing the "skimmability" of these documents, should we shorten or eliminate this?  Will we end up with more repetition across these documents?

Reply:
Will change to: "This example may also be relevant to the following:"